fip-0026.md

```diff
@@ -339,251 +409,281 @@ This sets the marketplace configurations.
     "owner_public_key": "FIO6TJcqTGShkALUSdAyW4rhRt1kjgz5PuVUyf4pmaEEeb7VtCRy3",
     "comissionfee": "6",
     "listingfee": "3",
-    "duration": "90",
-    "warn_days": "10",
+    "e_break": 0,
+    "max_fee": 1000000000,
+    "tpid": "rewards@wallet",
+    "actor": "rampayer"
 }
 ```
 #### Processing
 - Request is validated per exception handling
 - Write data to marketplace config table
+- Verify that the fee for this does not exceed the max fee specified.
+- Charge appropriate fee (this will be a bundled fee transaction, fee will be same as reject)
+- Verify tx does not exceed max transaction size.
 
 #### Exception handling
 |Error condition|Trigger|Type|fields:name|fields:value|Error message|
 |---|---|---|---|---|---|
-|Invalid marketplace name|Supplied `marketplace` name is not between 1 and 25 characters|400|marketplace| |"Length of marketplace name should be between 1 and 25 characters"|
-|Invalid actor name|Supplied `owner` is not a registered actor on FIO.|400| | |"Account not registered on FIO"|
-|Invalid public key|Invalid FIO Public Key|400| | |"Invalid Public Key"|
-|Invalid commission fee|`commissionfee` is not between 0-25|400| | |"Commission fee can only be between 0 and 25"|
-|Invalid listing fee|`listingfee` is not between 0-10|400| | |"Listing fee can only be between 0 and 10"|
-|Invalid Duration|`duration` is not a positive number less than 365|400| | |"Duration can only be between 1 and 365"|
-|Invalid Warn Days|`warn_days` must be a positive number less than the value of `duration`|400| | |"`warn_days` must be a positive number less than the value of `duration`"|
+|Invalid marketplace name|Supplied `marketplace` name is not between 1 and 25 characters|400|"marketplace"|empty string, "", "areallylongmarketplacename" |"Length of marketplace name should be between 1 and 25 characters"|
+|Invalid actor name|Supplied `owner` is not a registered actor on FIO.|400|"owner"| |"Account not registered on FIO"|
+|Invalid public key|Invalid FIO Public Key|400|"owner_public_key"|Value sent in, e.g."FIOinValidKey1234232"|"Invalid Public Key"|
```

Comment:
Need to define somewhere what happens if the public key is valid, but not an account.